ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/port_profiles.schema.yml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  port_profiles:
+    type: list
+    documentation_options:
+      filename: Connected Endpoints
+    primary_key: profile
+    convert_types:
+      - dict
+    description: |
+      Optional profiles to share common settings for connected_endpoints and/or network_ports.
+      Keys are the same used under endpoints adapters. Keys defined under endpoints adapters take precedence.
+    items:
+      type: dict
+      keys:
+        profile:
+          type: str
+          description: Port profile name.
+        parent_profile:
+          type: str
+          description: |
+            Parent profile is optional.
+            Port_profiles can refer to another port_profile to inherit settings in up to two levels (adapter->profile->parent_profile).
+        # "$ref": "#/keys/connected_endpoints/items/keys/adapters"
```

Comment:
@ClausHolbechArista @carlbuchmann Can you possibly help with this ref? I'm trying to inherit all the keys under adapters within the port_profiles schema. There is also a key called profile in connected_endpoints.adapters, do you see this being a possible issue?